### PR TITLE
Task/add fns for sorted sets

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add functions for dealing with sorted sets

### CODE REVIEW

File: notifications_utils/clients/redis/redis_client.py [hunk 1]
```diff
@@ -130,6 +129,64 @@ def exceeded_rate_limit(self, cache_key, limit, interval, raise_exception=False)
...
+        :param raise_exception:
+        """
+        cache_key = prepare_value(cache_key)
+        if self.active:
+            try:
+                self.redis_store.zremrangebyscore(cache_key, min_score, max_score)
+            except Exception as e:
+                self.__handle_exception(e, raise_exception, "delete-key-from-ordered-set", cache_key)
+
+    def get_length_of_sorted_set(self, cache_key, min_score=None, max_score=None, raise_exception=False):
```
Comment (jimleroyer) [inline notifications_utils/clients/redis/redis_client.py:164]: We don't want to follow previous pattern set by the `exceeded_rate_limit` function?

**Review by jimleroyer — COMMENTED** [review]
A few questions on how the new code will get used.

File: notifications_utils/clients/redis/redis_client.py [hunk 2]
```diff
@@ -130,6 +129,64 @@ def exceeded_rate_limit(self, cache_key, limit, interval, raise_exception=False)
...
+        """
+        Get the length of a sorted set. If we pass in min and max values, we delete the keys in that range.
```
[the remark this answers is not in the document]
Comment (jzbahrai) [reply notifications_utils/clients/redis/redis_client.py:166]: I was going to pass in explicit details min score = epoch - 24 hours, max score = epoch

File: notifications_utils/clients/redis/redis_client.py (hunk 1 above)
Comment (jzbahrai) [reply notifications_utils/clients/redis/redis_client.py:164]: so the exceed_rate_limit function was doing both the add to the sorted list + returning the length of the sorted list. We will need these to be different functions hence the split

**Review by jimleroyer — APPROVED** [review]
LGTM!

**Pull request merged**